install.bat

```diff
@@ -36,7 +36,7 @@ ECHO @ECHO OFF > %ChocolateyInstall%\bin\xslt-config.bat
 
 SET XSLT_INCLUDE=%ChocolateyInstall%\lib\xsltproc\dist\include
 SET XSLT_LIB_DIR=%ChocolateyInstall%\lib\xsltproc\dist\lib
-SET RUBY_BIN=c:\tools\ruby25\bin
+SET RUBY_BIN=c:\tools\ruby23\bin
```

Comment:
I'm not happy about this solution, I looking for some `choco` command which can get executable path from package name, in progress